server/app.js

```diff
@@ -19,5 +20,14 @@ app.use('/doc', swaggerUI.serve, swaggerUI.setup(swaggerDoc));
 app.use('/api/v1', router);
 
 const PORT = process.env.PORT || 3000;
-app.listen(PORT);
+
+const dbconn = new Db();
```

Comment:
The Db connection should be in the mainApp.js

Reply:
I included it in this part for debugging. so that I can know which DB am running on. But I will clean it later.